CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
-### 0.3.3 (Next)
+### 0.3.4 (Next)
```

Comment:
Leave the version alone, 0.3.3 has not been released and so your changes should just live under next.